index.html

```diff
@@ -21,12 +21,20 @@
       <a href="" id="logo">Ikonde Arnold Nekemiah</a>
     </div>
     <nav class="dropdown">
-      <img src="./Images/Icon - Menu.png" alt="">
+        <img class="menu" src="./Images/Icon - Menu.png" alt="">
```

Comment:
[OPTIONAL] Kindly make sure your logo aligned to the middle vertically like on the Figma template.
<img width="399" alt="Screenshot 2023-03-04 at 2 01 21 PM" src="https://user-images.githubusercontent.com/25887253/222896525-3e7e139a-4503-42e6-941f-c71a98cb88a5.png">
<img width="399" alt="Screenshot 2023-03-04 at 2 02 07 PM" src="https://user-images.githubusercontent.com/25887253/222896527-c211c789-eb45-4bf2-8028-04fe5034b16f.png">

-[OPTIONAL] Kindly make sure the space between nav links matches with the Figma template.
<img width="399" alt="Screenshot 2023-03-04 at 2 01 31 PM" src="https://user-images.githubusercontent.com/25887253/222896560-de54db2d-575f-43aa-8c99-2f7de925c6ac.png">